### DISCUSSION TITLE
In example plugin, where does com.example.opendj.server.ExamplePluginCfg come from?

### DISCUSSION THREAD

**Discussion by JJWTimmer** [OpenIdentityPlatform/OpenDJ#288] [Q&A] [answered]
Hi,
Looking at https://github.com/OpenIdentityPlatform/OpenDJ/blob/master/opendj-server-example-plugin/src/main/java/com/example/opendj/ExamplePlugin.java

I see import com.example.opendj.server.ExamplePluginCfg;
but I cannot find it.

What is it? 

Thanks!

**Comment by maximthomas** [comment, marked as answer]
Hi,
This file is generated, please check in `OpenDJ/opendj-server-example-plugin/target/generated-sources/config/com/example/opendj/server` folder after maven build
